### PR TITLE
Add events docs template + logic

### PR DESCRIPTION
**Pull request by UrbyK**
Please take a look.

### CODE REVIEW

File: pkg/codegen/events.go [hunk 1]
```diff
@@ -156,6 +156,8 @@ func genEvents(tpl *template.Template, dd ...*eventsDef) (err error) {
 		// Always regenerated
 		tplEventsGen = tpl.Lookup("events.gen.go.tpl")
 
+		// tplEventsAdoc = tpl.Lookup("events.gen.adoc.tpl")
```
Comment (tjerman) [inline pkg/codegen/events.go]: Remove this

File: pkg/codegen/events.go [hunk 2]
```diff
@@ -203,3 +205,32 @@ func makeEventGroup(pfix string, ee []string) (out []string) {
...
+		dst string
+	)
+
+	dst = path.Join(docsPath, "events_env_variables.gen.adoc")
+	return plainTemplate(dst, tplEventsAdoc, map[string]interface{}{
+		"Definitions": dd,
+		"Import":      collectEventsDefImports("", dd...),
+	})
+
+	//	return nil
```
Comment (tjerman) [inline pkg/codegen/events.go]: Remove this

File: pkg/codegen/events.go (hunk 1 above)
Comment (darh) [inline pkg/codegen/events.go]: Remove.

**Review by tjerman — CHANGES_REQUESTED** [review]
Please also open up a PR for the docs

File: pkg/codegen/events.go [hunk 3]
```diff
@@ -203,3 +205,32 @@ func makeEventGroup(pfix string, ee []string) (out []string) {
...
+	var (
+		tplEventsAdoc = tpl.Lookup("events.gen.adoc.tpl")
+
+		dst string
+	)
+
+	dst = path.Join(docsPath, "events_env_variables.gen.adoc")
+	return plainTemplate(dst, tplEventsAdoc, map[string]interface{}{
+		"Definitions": dd,
+		"Import":      collectEventsDefImports("", dd...),
```
Comment (darh) [inline pkg/codegen/events.go]: like mentioned on our call - this was added by mistake. remove the line and the function bellow as well.

**Review by darh — CHANGES_REQUESTED** [review]

**Pull request merged**